docs/sources/next/using-k6-browser/migrating-to-k6-v0-52.md

```diff
@@ -275,7 +275,7 @@ Below is a screenshot of a comparison between a generic browser test in `v0.51`
 
 ## Working with k6 check
 
-The k6 `check` API will not `await` promises, so calling a function that returns a `Promise`, for example `locator.textContent()`, inside one of the predicates will not work. Instead you will have to `await` and store the result in a variable _outside_ the `check`:
+The k6 `check` API will not `await` promises, so calling a function that returns a `Promise`, for example `locator.textContent()`, inside one of the predicates will not work. Instead you can work with [the jslib.k6.io's `check` utility](https://grafana.com/docs/k6/latest/javascript-api/jslib/utils/check/).
```

Comment:
```suggestion
The k6 `check` API will not `await` promises, so calling a function that returns a `Promise`, for example `locator.textContent()`, inside one of the predicates will not work. Instead, you can work with [the jslib.k6.io's `check` utility function](https://grafana.com/docs/k6/latest/javascript-api/jslib/utils/check/).
```

Reply:
I've also updated the previous versions: [3c22591](https://github.com/grafana/k6-docs/pull/1746/commits/3c22591877f68479868f3c459da1b7eda1d4208a)